### ISSUE TITLE
Cannot tab to Cookiecutter template description

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2241]
As a result, screen reader cannot read it.

**Comment by zooba** [comment]
There's no action to take on this item, so tabbing to it is the wrong action.

Need more information on how to enable screen readers to read this text without breaking keyboard users. @eclectir any ideas?

**Comment by zooba** [comment]
I think I know how to do this - it should be a `Label`, not a `TextBlock` (or a Label containing a TextBlock)

**Comment by zooba** [comment]
You need to use Caps+Left/Right arrow to get to it, not Tab.
